package.json

```diff
@@ -90,6 +90,10 @@
       {
         "name": "2021-09-release",
         "prerelease": true
+      },
+      {
+        "name": "intent-api",
+        "prerelease": true
```

Comment:
Can CI handle double prerelease @derberg? 🤔

Reply:
yes, it is ready as it was required for React component and the 2.0 prerelease phase with `next` branch. 

We only need to agree on bumping strategy. In React for example every single release candidate goes to `html-template`, so on master we had to make sure that whenever you release to `master`, main release is not bumped in `html-template` -> https://github.com/asyncapi/asyncapi-react/blob/master/.github/workflows/bump.yml#L32

Reply:
@jonaslagoni but I guess we do not want to bump it automatically anywhere. I don't see a point at this moment

Reply:
all templates follow guidelines to set that they are always compatible with generator prior 2.0
so yeah, pushing new Parser would be breaking change, unless you are thinking about release candidates in generator too 🤔

Reply:
I am 🙂

Reply:
well, I love the idea!
but 😄 I don't think my bumper supports custom branches 😆

Reply:
@smoya ping as not sure if you saw it. I think this one is not just nice to have, but tbh a must-have, otherwise testing new functions in templates will be complicated

Reply:
@smoya @jonaslagoni good news, I just released v4 of the bumper action

https://github.com/derberg/npm-dependency-manager-for-your-github-org

Now it supports new prop for config - `base_branch`

So once we merge this PR, and create branch `next-major`, please open another PR against this new release branch and make sure that the bump.yml workflow:
- its version in `next-major` branch will react to commits to `next-major` and not `master`
- add `base_branch: next-major`. This will force bumper to bump Parser in dependants that also have `next-major` branch, and PR with bump will be opened against `next-major`, which means that after merge the release candidate will be published for a given project, if configured

makes sense?

Reply:
That's awesome! Yeah makes totally sense. Thanks for the work you did in the bumper action!
Let's do that thing you suggest then!

Reply:
It's done here: https://github.com/asyncapi/parser-js/pull/468 @derberg @jonaslagoni